civictechprojects/static/css/partials/_ProjectCard.scss

```diff
@@ -95,87 +95,3 @@
   padding: 0.25rem 0.5rem;
   margin: 0.25rem 0.5rem 0.25rem 0;
 }
-
```

Comment:
With the change to a single card layout after the homepage was designed, none of this media query is relevant - the rules for existing classes (such as .ProjectCard-root) and the otherwise completely unused classes (such as .ProjectCardOverlay)

Reply:
Removed in e862d5c